src/Featurizers/TestHelpers.h

```diff
@@ -138,6 +138,45 @@ bool FuzzyCheck(std::vector<T> const & vec1, std::vector<T> const & vec2, std::d
     }
     return true;
 }
+template<>
```

Comment:
We don't need to use a template here, as functional overloads will work just fine.